client/src/pages/public/EmployerLogin.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Page, Button } from '../../components/generic';
+import Box from '@material-ui/core/Box';
+import { makeStyles } from '@material-ui/core/styles';
+import { Routes } from '../../constants';
+import { useHistory } from 'react-router-dom';
+import InfoIcon from '@material-ui/icons/Info';
+
+import backgroundImage from '../../assets/images/employer_login_bg.png';
+
+const useStyles = makeStyles((theme) => ({
+  blueText: {
+    color: theme.palette.primary.light,
+  },
+  blueBox: {
+    backgroundColor: '#EDF6FF',
+    maxWidth: 554,
```

Comment:
That's a very specific number - any reason why? I'm wondering if there might be a woraround

Reply:
I copy this thing from the existing participant login page. So, I've no idea.